ApiDoctor.Validation/MethodDefinition.cs

```diff
@@ -127,6 +128,11 @@ public static MethodDefinition FromRequest(string request, CodeBlockAnnotation a
         /// </summary>
         public string Request {get; private set;}
 
+        /// <summary>
+        /// The parsed Request data from the documentation. 
+        /// </summary>
+        public HttpRequest HttpRequest { get; private set; }
```

Comment:
I think we can undo this change. It is only set in line 107 but never really used elsewhere.